src/QMCHamiltonians/NonLocalECPComponent.h

```diff
@@ -239,6 +244,24 @@ class NonLocalECPComponent : public QMCTraits
                                        const std::vector<ValueType>& dlogpsi,
                                        std::vector<ValueType>& dhpsioverpsi);
 
+  void evaluateOneBodyOpMatrixContribution(ParticleSet& P,
+                                           const int iat,
+                                           const TWFFastDerivWrapper& psi,
+                                           const int iel,
+                                           const RealType r,
+                                           const PosType& dr,
+                                           std::vector<ValueMatrix>& B);
+
+  void evaluateOneBodyOpMatrixdRContribution(ParticleSet& P,
```

Comment:
Document these two functions. Right now I have to read the code to understand the dimensions of B and dB.

Reply:
Oops.  Thank you!  I'll add that.